frontend/smart-kickers-game/src/App.css

```diff
@@ -15,16 +18,20 @@
 }
 
 html {
-  color: var(--html-color);
   font-family: 'lato', 'Arial', sans-serif;
   font-weight: 300;
   font-size: 20px;
   text-rendering: optimizeLegibility;
+  background-color: var(--background-color);
+  color: var(--font-color);
 }
 
-body {
-  color: var(--html-color);
-  background-color: var(--my-white);
+.whiteTeamIcon {
+  color: var(--font-color);
+}
+
+.blueTeamIcon {
+  color: var(--blue-icon);
 }
```

Comment:
Shouldn't those be defined in `GameStatistics.css`?

Reply:
We will change it.

Reply:
We've added icons also in the live game view so we will keep it as it is.